### DISCUSSION TITLE
Reverse the "Who Can Ignore this" Setting

### DISCUSSION THREAD

**Discussion by stevejburge** [publishpress/publishpress-checklists#466] [Feature Requests]
We have this the wrong way around.

This is should be positive - Which roles must complete this task?

**Comment by andergmartins** [comment]
"Which roles must complete this task" would probably clash with Custom tasks, which have a similar field, but for slightly different use case, I guess.

<img width="1110" alt="Screen Shot 2021-06-15 at 10 50 59" src="https://user-images.githubusercontent.com/407131/122064507-96601f80-cdc7-11eb-8f6a-70b148194758.png">

The first roles field there works for selecting which user roles can't publish without the task is completed. The second one (only available for custom tasks) is there to select which roles can mark the task as completed. So maybe we need to think about another way to name the field we are trying to refactor.

**Comment by stevejburge** [comment]
@andergmartins Good point. Let's keep this as it is for now

**Comment by julianactrl** [comment]
Hello Steve,
Sorry for bothering you.
Would you like to check your mail?
I think you missed my email, please check my email and let me know.
Thanks.
